src/interfaces/IChainlinkOracle.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+import {IERC4626} from "./IERC4626.sol";
+import {AggregatorV3Interface} from "./AggregatorV3Interface.sol";
+import {IOracle} from "../../lib/morpho-blue/src/interfaces/IOracle.sol";
+
+interface IChainlinkOracle is IOracle {
```

Comment:
Consider following the same best practice you have already used in other interface contracts by covering also the interface with full natspec doc tags. 

Consider adding at least @title and @notice (to describe the interface)

Reply:
To be clear, I mean something along the line of

```solidity
/// @title IChainlinkOracle
/// @author Morpho Labs
/// @custom:contact security@morpho.org
```

Reply:
Fixed here #55